tests/test_rtdc_writer.py

```diff
@@ -66,8 +66,10 @@ def test_bulk_image():
         events = rtdc_data["events"]
         assert "image" in events.keys()
         assert np.allclose(events["image"][6], image[6])
-
-
+        image = rtdc_data["events"]['image']
+        assert b'IMAGE' in image.attrs.get('CLASS')
```

Comment:
You should also be checking the shape of the image to make sure it is correct.

Reply:
Yes, added shape too

Reply:
> Yes, added shape too

Sorry, I thought this is in another test function.

Reply:
Now, I have updated all these tests in order to check shape of the feature. Please find them in this commit https://github.com/ZELLMECHANIK-DRESDEN/dclab/pull/172/commits/08c8d2097d24f8c1eef7242ed8d27cd283b09300